doc/cephadm: Document mgr/cephadm/manage_etc_ceph_ceph_conf (#36796)
ceph/ceph: doc/cephadm/operations.rst

/etc/ceph/ceph.conf
===================

Cephadm distributes a minimized ``ceph.conf`` that only contains 
a minimal set of information to connect to the Ceph cluster.

> To update the configuration settings, instead of manually editing
> the ``ceph.conf`` file, use the config database instead::

Reviewer: ```suggestion
Use the config database to update the configuration settings,
instead of manually editing the ``ceph.conf`` file::
```
minor reword to avoid using `instead` multiple times

  ceph config set ...

See :ref:`ceph-conf-database` for details. 

> By default, cephadm does not deploy that minimized ``ceph.conf``

Reviewer: ```suggestion
By default, cephadm does not distribute a minimized ``ceph.conf``
```

> across the cluster. To enable the management of 
> ``/etc/ceph/ceph.conf``files on all hosts, please enable this 
> by running::

Reviewer: ```suggestion
across all hosts in the cluster. This can be enabled by running::
```
less is more?
